### PR TITLE
pornolab: migrate to yaml

### PR DESCRIPTION
**Pull request by mynameisbogdan**
Works okay in YAML with the download block logic removed since the download is present on the results page, but the main reason being captcha support for use in Prowlarr.

* https://github.com/Prowlarr/Prowlarr/issues/1258

### CODE REVIEW

File: src/Jackett.Common/Definitions/pornolab.yml [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+    seeders:
+      selector: td.seedmed > b
+    leechers:
+      selector: td.leechmed > b
+    grabs:
+      selector: td:nth-child(9)
+    downloadvolumefactor:
+      text: 0
+    uploadvolumefactor:
+      text: 1
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/pornolab.yml:252]:
```suggestion
    downloadvolumefactor:
      text: 1
    uploadvolumefactor:
      text: 1
    minimumratio:
      text: 0.31
```

C# indexer has dlvf as `1`, and https://github.com/Jackett/Jackett/issues/9456 states there's no freeleech, which seems to be the case from a quick browse.

[Rule 4.5](https://pornolab.net/forum/viewtopic.php?t=980423) also says there's a global minimum ratio of `>0.3`, so we could add that as well.

**Review by ilike2burnthing — CHANGES_REQUESTED** [review]
Haven't tested yet but looks good, just 2 minor suggestions.

**Comment by garfield69** [comment]
the reason this was written in c# to begin with was because of https://github.com/Jackett/Jackett/issues/4373#issuecomment-455869982
this may no longer be relevant or perhaps may never have been necessary but it should be checked before committing.

**Comment by ilike2burnthing** [comment]
Everything seems to be working ok, including downloads, although I ran into a weird bug:

- configure indexer successfully
- attempt to reconfigure indexer
- login fails - login.php redirects to index.php
- restart Jackett
- reconfigure indexer successfully
- attempt to reconfigure indexer
- login fails

As far as I can tell, Jackett is being seen as already logged in and so can't access the login page if done in the same session.

This doesn't affect the C# indexer, and I can't see a difference in what we're doing to login with the YAML version, so perhaps some weird edge case issue with Cardigann?

**Pull request merged**